### DISCUSSION TITLE
I can't verify my student Status - UOC

### DISCUSSION THREAD

**Discussion by mcaloto** [community/community#51130] [GitHub Education]
No matter how hard I try, I am not able to get the system to verify my student status, I have completed my profile (photo included), I have uploaded my student card with expiration date, and I have also tried with the registration form and always I get rejected after a few minutes with the following message:

_`You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.`_

**Comment by samus-aran** [comment]
Hi @mcaloto 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We also suggest that after updating your profile information that you log out of GitHub and log back in before re-applying. Let us know how you get on 👍

**Comment by lluiscab** [comment]
I've been trying for the past hour to verify my student status, I keep getting `Unable to submit your application. Please try again
` errors all the time, I've tried sending my last tuition invoice and a digital certificate of academic enrollment issues by the UOC and nothing is accepted. Now I see that support won't even look manually at my application and I have to get help from the "Community". 

Why is this so hard? Why can't I upload PDF files? Why do I have to take screenshots of such files, edit them to match some random minimum width and height, save them in JPG (don't know what you guys hate jpg so much) and all of that for the machine to think the document I've submitted is an acceptance letter instead of a certificate of enrollment.

I'm even receiving the "unable to verify your academic affiliation" emails on my university's email address @uoc.edu.
